src/System.Security.Cryptography.Cng/src/Resources/Strings.resx

```diff
@@ -159,6 +159,12 @@
   <data name="Cryptography_InvalidCipherMode" xml:space="preserve">
     <value>Specified cipher mode is not valid for this algorithm.</value>
   </data>
+  <data name="Cryptography_InvalidCurve" xml:space="preserve">
```

Comment:
Sure. I suppose InvalidCurveName would have been a better choice than InvalidCurve, but I took your suggestion of CurveNotSupported